includes/payments/actions.php

```diff
@@ -302,6 +298,22 @@ function edd_record_order_status_change( $old_status, $new_status, $order_id ) {
 	}
 }, 10, 3 );
 
+/**
```

Comment:
@cklosowski I _finally_ remembered that we have the transition hooks in 3.0 so I've removed most of the calls to recalculate and made this new function. I haven't named it for now but I think this is way better than going through arrays of cart details (which was causing some issues during the purchase process anyway while order items were still pending). I'm wondering if we have need of an `order_items` directory with its own functions/actions file(s).

Reply:
Yeah @robincornett lets go ahead and create a new file to hold all the transitions related to payments. That will make things much easier to maintain later on.